ivy/functional/ivy/elementwise.py

```diff
@@ -3319,36 +3320,22 @@ def round(
     ivy.array([[0.,5.,-343.,2.],[-6.,44.,12.,12.]])
 
     With :code:`ivy.NativeArray` input:
-    >>> x = ivy.NativeArray([20.2, 30.5, -5.81])
+
+    >>> x = ivy.native_array([20.2, 30.5, -5.81])
     >>> y = ivy.round(x)
     >>> print(y)
     ivy.array([20.,30.,-6.])
 
     With :code:`ivy.Container` input:
+
     >>> x = ivy.Container(a=ivy.array([4.20, 8.6, 6.90, 0.0]),\
                   b=ivy.array([-300.9, -527.3, 4.5]))
     >>> y = ivy.round(x)
     >>> print(y)
-    {a:ivy.array([4.,9.,7.,0.]),b:ivy.array([-301.,-527.,4.])}
-
-    Instance Method Examples
-    ------------------------
```

Comment:
> Actually, I thought as they are included in the instance method's documentation there is no need to put them here on the functional side. I think it can be to just keep repeating the same things.

You are definitely right!